icons/signature.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "AndreasSas",
+    "jguddas"
+  ],
+  "tags": [
+    "text",
+    "format",
+    "input"
```

Some suggestions for more tags.
```suggestion
    "input",
    "contract",
    "autograph",
    "handwriting",
    "sign",
    "cursive",
    "ink",
    "scribble",
    "authorize",
    "personal",
    "agreement",
    "legal",
    "document",
    "identity",
    "authentic",
    "approval",
    "verification",
    "unique",
```